psasim/include/client.h

```diff
@@ -14,7 +14,10 @@ extern "C" {
 
 #include <stdint.h>
 #include <stddef.h>
-#include <psa/error.h>
+
```

Comment:
Please use [`mbedtls-move-to-framework‎`](https://github.com/Mbed-TLS/mbedtls-docs/pull/145) instead of importing a rewritten history. With a rewritten history, we don't get the same commits.